.eslintrc.json

```diff
@@ -0,0 +1,21 @@
+{
+  "extends": "react-app",
+	"ignorePatterns" :["dist/*"],
+	"env": {
+		"browser": true,
+		"node": true
```

Comment:
Setting node env seems counterintuitive. Removing it doesn't seem to affect the linting at all, so we can disable it, I assume.

Reply:
Could you explain why it is counterintuitive? I added this because it was present in the previous version of this file.

Reply:
radio is a web app, so we don't want eslint to bring any node globals into scope while linting. This makes linting and building inconsistent with each other, which isn't great.

Reply:
I will remove it